### ISSUE TITLE
Shorter/Custom Tab Name

### ISSUE THREAD

**Issue by kedepot** [makehumancommunity/mpfb2#104]
Enhancement/Suggestion: 
Please remove the version number from the N-panel name: The tab becomes very long, and for a lot of people, n-panel space is already cramped. Alternatively, make the the name configurable in the add-on prefs. Or, the version nr can also be added to a panel header instead.

![tabname](https://user-images.githubusercontent.com/95410139/236764485-0353d3a7-6644-4f8f-ba3b-523c5743213a.png)

**Comment by joepal1976** [comment]
I've pushed an update now, providing a way for overriding the shelf label. Once you get the latest code either from github or from tomorrows nightly build, you'll find it in the preferences window:

![image](https://github.com/makehumancommunity/mpfb2/assets/10105480/3a3a1ed9-03f8-4bfe-9057-e41b3f774eda)

**Comment by ashledombos** [comment]
Strange because I don’t see it working. I did restart Blender, just in case.

![image](https://github.com/makehumancommunity/mpfb2/assets/513335/d3a00759-ccb6-4298-aaad-6ef085498311)

and the _preferences.py file seems correct

![image](https://github.com/makehumancommunity/mpfb2/assets/513335/00752bda-c453-4093-ba28-f96a4b66723c)

**Comment by ashledombos** [comment]
OK, I found the culprit. It seems there is a severe incompatibility with Clean Panels plugin. Once I deactivate it, the renaming feature is working.
I notice that Clean Panels also create some additional mess with MPFB, as it changes the order of sections. 

![image](https://github.com/makehumancommunity/mpfb2/assets/513335/f8d2d730-4eb7-460b-82fd-a80fd4dcc14d)

I guess there may be other issues I did not notice yet, then.
